api/OfflineAudioContext.json

```
@@ -453,12 +477,26 @@
             "opera_android": {
               "version_added": "14"
             },
-            "safari": {
-              "version_added": "6"
-            },
-            "safari_ios": {
-              "version_added": null
-            },
+            "safari": [
+              {
+                "version_added": "14.1"
+              },
+              {
+                "version_added": "6",
+                "version_removed": "14.1",
+                "prefix": "webkit"
+              }
+            ],
+            "safari_ios": [
+              {
+                "version_added": "14.5"
+              },
+              {
+                "version_added": "≤10.3",
```

```suggestion
                "version_added": "6",
```

Confirming the data for `OfflineAudioContext` proved difficult, but this is consistent with the current claim that `webkitOfflineAudioContext` was introduced in iOS 6. Using http://mdn-bcd-collector.appspot.com/tests/api/webkitOfflineAudioContext I'm actually able to confirm that *wasn't* the case, and that it must have shipped either in 6.1 / 7.0 or 6.2 / 7.1. Unfortunately I don't have access to 6.1 / 7.0 and this runs into the complications in https://github.com/mdn/browser-compat-data/issues/9423, but just matching the base feature seems OK here. If that data is updated later, this lint would fail and this would have to change too.